src/mdm-data-class.model.ts

```diff
@@ -45,13 +45,13 @@ export type DataClassIndexResponse = MdmIndexResponse<DataClass>;
 export type DataClassDetailResponse = MdmResponse<DataClassDetail>;
 
 /**
- * Represents hierarchy information for a Data Class.
+ * Represents information for a Data Class relating to a hierarchy.
  */
-export interface DataClassHierarchy {
+export interface DataClassNode {
   /**
    * The child data classes under this Data Class.
    */
-  dataClasses?: DataClassHierarchy[];
+  dataClasses?: DataClassNode[];
```

Comment:
childDataClasses/childDataElements? would remove the need for the comment. and it's consistent with the naming in the DataModelNode interface.

Reply:
Sorry, missed the previous comment!